src/main.js

```diff
@@ -183,52 +197,48 @@ var manifestor = function(options) {
     if (!arguments.length) return _canvasImageStates;
     _canvasImageStates = state;
 
-    // if (!initial) {
-    //     jQuery.publish('annotationsTabStateUpdated' + this.windowId, this.tabState);
-    // }
-
     return _canvasImageStates;
   }
 
-  // function detailTransition(detailLayout) {
```

Comment:
Dead code that is never used, although in the refactor it would be nice to get to the point where the transitions have intelligible names.